src/core/simulation.jl

```diff
@@ -79,7 +80,18 @@ end
 get_ini_cond_chronology(s::Simulation, number::Int64) = get(s.sequence.ini_cond_chronology, s.sequence.order[number], nothing)
 
 
-
+function _check_inputs(sim::Simulation)
```

Comment:
This function should have a better name since it is only checking the chronology inputs to the simulation.